edx_lint/files/pylintrc

```diff
@@ -36,28 +249,88 @@ load-plugins=edx_lint.pylint,pylint_django,pylint_celery
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
 disable=
+    # These should be left to the discretion of the reviewer
+    bad-continuation,
```

Comment:
Is this one we regularly suppress?  It seems like it should be pretty straightforward to appease.